app/controllers/api/base_controller.rb

```diff
@@ -4,6 +4,7 @@
 
 module Api
   class BaseController < ActionController::Metal
+    include ActionController::StrongParameters
```

Comment:
wonderful, I am happy I raised this point!
I hope we can keep it in the rails 4.2 branch with the responders thing :+1:
do we need responders in the api, why? just asking.

Reply:
Everything is broken if responders are missing. It's the logic that handles the response that's given by an action, like returning an object in json.

Reply:
Hopefully we can include it in `Metal` without too much hassle :+1:

Reply:
The build looks like this, in case you're interested: https://semaphoreci.com/openfoodfoundation/openfoodnetwork-2/branches/rails-4-2-rebased/builds/7

Absolute carnage!

Reply:
ehhee, it's only one error, right?
JSON text must at least contain two octets.

Reply:
Ahahah, yes. Only one small error: none of the endpoints give a response at all. :rofl: 

I think it's fixed now in the Rails 4.2 branch with `Metal` :tada: